src/plugins/shared_ux/public/components/redirect_app_links/redirect_app_links.stories.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiButton } from '@elastic/eui';
+import React from 'react';
+import { BehaviorSubject } from 'rxjs';
+
+import { action } from '@storybook/addon-actions';
+import { RedirectAppLinks } from './redirect_app_links';
+import mdx from './redirect_app_links.mdx';
+
+export default {
+  title: 'Redirect App Links',
+  description: 'app links component that takes in an application id and navigation url.',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Component = () => {
+  return (
+    <RedirectAppLinks
+      navigateToUrl={() => Promise.resolve()}
+      currentAppId$={new BehaviorSubject('test')}
+    >
+      <EuiButton
+        data-test-subj="storybookButton"
+        fill
+        iconType="plusInCircle"
+        onClick={action('button pressed')}
+      />
```

Comment:
The button still needs a label/text it just didn't need to be internationalized 😄 
<img width="162" alt="Screen Shot 2022-03-10 at 16 59 16 PM" src="https://user-images.githubusercontent.com/549577/157761878-a02d8220-38c3-4134-bbb8-4a21f94a8370.png">
```suggestion
      >Test link</EuiButton>
```

Reply:
Thank you~ [2d2ddc1](https://github.com/elastic/kibana/pull/124197/commits/2d2ddc1a2e57056ae6853c3e9ee2c7c1f4a8bd07) for changes and for below comment as well :face_palm: